### PR TITLE
How about another spec?

### PR DESCRIPTION
**Pull request by jwoertink**
We have a bug we're trying to track down. Thought it was in the router, but it seems this works. I figure since another spec couldn't hurt, why not just add this in? 😂

### CODE REVIEW

File: spec/integration_spec.cr [hunk 1]
```diff
@@ -33,6 +34,7 @@ describe LuckyRouter do
       router.match!("get", "/users/1/new").payload.should eq :new
       router.match!("get", "/users/1/tasks/1").payload.should eq :user_tasks
       router.match!("get", "/admin/users/1/tasks/1").payload.should eq :admin_user_tasks
+      router.match!("get", "/admin/reports/1.csv").payload.should eq :admin_reports
```
Comment (paulcsmith) [inline spec/integration_spec.cr]: I'm worried that the param name key will be `"report_id.csv"` and not just `report_id`. Could you add a spec that makes sure the param payload is correct?

Comment (jwoertink) [reply spec/integration_spec.cr]: Yup. good call. It would be `report_id.csv`. So looks like I may need to update some actual code then 😝

**Comment by paulcsmith** [comment]
Now that you’re trying out the new approach, is this still necessary/helpful?

**Comment by jwoertink** [comment]
Actually no. I'll close this out. However, we will need to have some docs mentioning that when you create a route, you don't ever specify the file extension.

**Pull request closed without merging**